src/screens/conclusion/index.js

```diff
@@ -1,73 +1,85 @@
-import "./conclusion.css";
+import { PatreonIcon } from "../../components/patreon-icon";
 
 export function Conclusion() {
   return (
-    <section id="conclusion" className="together-we-can screen-content">
-      <div className="full-screen bears h-full w-full flex-col justify-between">
-        <div className="h-screen-50vh flex w-[100%] flex-col self-start py-16 pr-10 lg:w-[75%] 2xl:w-[50%]">
-          <h2>Together, we can make life better for wild animals.</h2>
+    <section
+      id="conclusion"
+      className="relative min-h-screen bg-woods bg-cover md:bg-woods-md md:bg-center"
+    >
+      <img
+        className="absolute right-20 top-52 w-96 lg:w-[500px]"
+        src="/images/conclusion/momma.png"
+        alt="Mother bear"
+      />
+      <img
+        className="absolute bottom-72 right-96 w-[500px]"
+        src="/images/conclusion/pups.png"
+        alt="Bear pups"
+      />
+      <div className="py-30 flex max-w-3xl flex-col pt-36 pl-28 text-white">
+        <h2 className="z-10 mb-4 font-brand text-5xl leading-relaxed">
+          Together, we can make life better for wild animals.
+        </h2>
+        <p className="font-lg z-10 font-sans">
```

Comment:
These `z-10`s can be removed, right? You can apply a `relative` class to the surrounding div here if the images are being displayed on top without `z-10`. Adding `relative` creates a new stacking context that solves this issue.

`font-sans` should be set by default, so I think that can also be removed